app/client/src/workers/Evaluation/evaluate.ts

```diff
@@ -365,15 +366,16 @@ export function setEvalContext({
   Object.assign(self, evalContext);
 }
 
-export default function evaluateSync(
+export function evaluateSync(
   userScript: string,
   dataTree: DataTree,
   isJSCollection: boolean,
   context?: EvaluateContext,
   // TODO: Fix this the next time the file is edited
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   evalArguments?: Array<any>,
-  configTree?: ConfigTree,
+  configTree: ConfigTree = {},
+  scopeCache?: EvalContext,
```

Comment:
Can we name the variable to be able to explain it's usecase. 
```suggestion
  evalContextCache?: EvalContext,
```